PodTemplates.d/package-windows

```diff
@@ -22,11 +22,11 @@ spec:
     name: "jnlp"
     resources:
       limits:
-        memory: "1Gi"
-        cpu: "1"
+        memory: "8Gi"
+        cpu: "2"
       requests:
-        memory: "1Gi"
-        cpu: "1"
+        memory: "8Gi"
```

Comment:
does it really need to request 8gi? that's a lot ? will need quite big vms for that

Reply:
There is already a windows node deployed as defined [here](https://github.com/jenkins-infra/azure/blob/master/plans/publick8s.tf#L128L142), last time I checked it wasn't possible to define a node pool with default value set to 0. In the futur we can reduce this limit according the need